vaadin-testbench-shared/src/main/java/com/vaadin/testbench/ElementQuery.java

```diff
@@ -48,33 +57,153 @@ public class ElementQuery<T extends TestBenchElement> {
     private static final int DEFAULT_WAIT_TIME_OUT_IN_SECONDS = 10;
 
     public static class AttributeMatch {
-        private final String name;
-        private final String operator;
-        private final String value;
-        private final boolean negate;
 
-        public AttributeMatch(String name, String operator, String value) {
-            this.name = name;
+        public enum Operator {
+            /**
+             * Attribute exists (with or without a value).
+             */
+            EXISTS(""),
+            /**
+             * Attribute value and given value match exactly.
+             */
+            MATCHES_EXACTLY("="),
+            /**
+             * Attribute value contains the given value.
+             */
+            CONTAINS("*="),
+            /**
+             * Attribute value contains a space-separated token
+             * that matches the given value.
+             */
+            CONTAINS_TOKEN("~="),
```

Comment:
nit: Just a personal preference: what about `CONTAINS_ITEM` or `CONTAINS_WORD`?

For example, I've found this description for the selector, using `word` term

> Represents elements with an attribute name of attr whose value is a whitespace-separated list of words, one of which is exactly value.

Reply:
Sure.